epub33/core/index.html

```diff
@@ -691,277 +732,462 @@ <h4>Conformance Checking</h4>
+							used without restriction in EPUB or Foreign Content Documents. For more information about
+							Core Media Type Resources, refer to <a href="sec-core-media-types"></a>.</p>
+
+						<div class="note">
+							<p>Being a Core Media Type Resource does not mean that Reading Systems will always render
+								the resource, as not all Reading Systems support all features of EPUB 3. A Reading
+								System without a <a>Viewport</a>, for example, will not render visual content such as
+								images.</p>
+						</div>
+
+						<p>The opposite of Core Media Type Resources are Foreign Resources. These are resources that
+							Reading Systems are not guaranteed to support the rendering of. As a result, similar to how
+							using Foreign Content Documents in the spine requires fallbacks to ensure their rendering,
+							using Foreign Resources in content documents also requires fallbacks. These fallbacks are
+							provided in one of two ways: using the capabilities of the parent format or via manifest
 							fallbacks.</p>
 
-						<p>The columns in the following table represent the following information:</p>
+						<p>The preferred method is to use the fallback capabilities of the parent format. Many HTML
+							elements, for example, have intrinsic fallback capabilities. One example is the <a
+								href="html#the-picture-element"><code>picture</code> element</a> [[HTML]], which allows
+							EPUB Creators to specify multiple alternative image formats.</p>
+
+						<p>If an intrinsic fallback method is not available, it is also possible to use manifest
+							fallbacks, but this method, as <a href="#caution-fallbacks">cautioned against</a> in the
+							previous section, is discouraged. For more information about Foreign Resources, refer to <a
+								href="#sec-foreign-resources"></a>.</p>
+
+						<p>EPUB 3 also carves out some special exceptions to the above resource requirements:</p>
 
 						<ul>
-							<li>
-								<p><strong>Media Type</strong>—The MIME media type [[RFC2046]] used to represent the
-									given Publication Resource in the <a href="#sec-manifest-elem">manifest</a>.</p>
-								<p>If the table lists more than one media type, the first one is the preferred media
-									type. EPUB Creators should use the preferred media type for all new EPUB
-									Publications.</p>
-							</li>
-							<li><strong>Content Type Definition</strong>—The specification to which the given Core Media
-								Type Resource must conform.</li>
-							<li><strong>Applies to</strong>—The Publication Resource type(s) that the Media Type and
-								Content Type Definition applies to.</li>
+							<li>Video resources are neither Core Media Type Resources nor Foreign Resources. EPUB
+								Creators may include any video format without fallbacks, but Reading Systems are not
+								guaranteed to support any video format.</li>
+							<li>Font resources do not require fallbacks because CSS rules will always ensure Reading
```

Comment:
Or do we say that fonts _MUST NOT_ have fallbacks? I don't think so...

Reply:
> Technically, no. Not for all font resources, only the ones listed in the CMT section.

Ie, those whose media type is listed. I got that, but maybe it is worth emphasizing it then... I will not be the only one asking this questions

(I thought that the four font types listed cover the vast majority of font types, so the leftover are not necessarily relevant in practice, but I may be very wrong...)

Reply:
Thinking about it further: the formulation of those four exceptions is still bothering me. I wonder whether it is not cleaner to say that, on the Content Document Axis (as it is called now) the categorization between Core Media Type and Foreign Resources are not exhaustive, but the division along that axis is three-way: CMT, Foreign, and, say (just for the sake of this comment) Extras. Extras are characterized by the fact that fallbacks are not to be defined for these files for various reasons. These include then videos, files that are linked or used via tracks, and fonts. It sounds different from 'exceptions'... Just a thought.

Reply:
Ya, we technically have a kind of "Exempt Resources" class that are neither CMTs not Foreign Resources. That's what video falls into, plus fonts anything in the list of exempt elements.

> I thought that the four font types listed cover the vast majority of font types, so the leftover are not necessarily relevant in practice

Probably, but do we know for certain no one has used any others? Do we know that no one had to use fallbacks to work around our not having a font media type in the CMT list in the past. If we explicitly disallow fallbacks for fonts we're potentially invalidating all these cases.

Still, I don't understand how we can exempt all fonts from fallbacks and also have a set of CMTs. My gut says if we need to explain what fonts are in the obfuscation section, we should list the table there.

Reply:
Although, before I get too hasty to change things, CMTs are supposed to be supported, so I guess technically we expect support for the CMT fonts while also allowing any other fonts without fallbacks even though there is no requirement of support for them. It's kind of a weird in-between situation. That's what's confusing me about the exemption.

Reply:
If we make an "exempt" class, though, we could move the font exemption explanation to a new section and explain its purpose better there. It would at least make clearer the distinction between fonts that Reading Systems are expected to support and ones they aren't.